### PR TITLE
Springboard notes for tests relocations and improvements

### PR DESCRIPTION
**Pull request by xandkar**
This PR for just notes. Each relocation will have its own PR.

### CODE REVIEW

File: test/miner_blockchain_SUITE.erl [hunk 1]
```diff
@@ -2,39 +2,91 @@
...
+    group_change_test,
+    %% RELOC TBD:
+    %%   FOR:
+    %%     - seems to be mostly calling a single miner
+    %%     - submits txns
+    %%   AGAINST:
+    %%     - seems to expect different outcomes in multiple miners
+
+    master_key_test,
+    %% RELOC MOVE - does things with vars and txns
```
Comment (xandkar) [inline test/miner_blockchain_SUITE.erl:76]: https://github.com/helium/blockchain-core/pull/944/

File: test/miner_blockchain_SUITE.erl [hunk 2]
```diff
@@ -2,39 +2,91 @@
...
+
+    version_change_test,
+    %% RELOC MOVE - does things with vars and txns
```
Comment (xandkar) [inline test/miner_blockchain_SUITE.erl:79]: https://github.com/helium/blockchain-core/pull/944/

File: test/miner_payment_v2_txn_SUITE.erl [hunk 3]
```diff
@@ -1,3 +1,4 @@
+%%% RELOC MOVE - not miner-specific functionality
```
Comment (xandkar) [inline test/miner_payment_v2_txn_SUITE.erl:1]: This looks redundant to existing payment tests in core. Maybe just remove.

Comment (evanmcc) [reply test/miner_payment_v2_txn_SUITE.erl:1]: I think that both payments tests can be removed, yes.

File: test/miner_txn_mgr_SUITE.erl [hunk 4]
```diff
@@ -1,3 +1,4 @@
+%%% RELOC MOVE? - looks like not miner-specific functionality
```
Comment (xandkar) [inline test/miner_txn_mgr_SUITE.erl:1]: Potentially re-use a single cluster and just use fresh addresses.

**Pull request merged**